heiko/migrate.py

```diff
@@ -1,18 +1,38 @@
 import sqlite3
 import getpass
 from heiko.utils import log
+import os
 
 
-def migrate_user(auth, client):
+def migrate_user(auth, client, cfgobj):
     """
     Migration function that reads data from old matomat sqlite and creates user in the backend
 
     :auth: dict
+    :client: userauth object
+    :cfgobj: config dict
     :returns: bool
     """
 
-    sq = sqlite3.connect('/home/heiko/matomat.db')
+    # Check if configured database is there and can be read
+    try:
+        migration_database = cfgobj["accouting"]["migration_database"]
+        if os.access(migration_database, os.R_OK):
+            matomat_db = sqlite3.connect(migration_database)
+        else:
+            raise IOError
+    except (IOError, KeyError):
+        log("Migration database cannot be found or is not configured. Aborting.", serv="ERROR")
+        return False
+
+    # First, ask for Admin yes/no
+    admin = input("Should the migrated user be admin? (y/n): ").lower()[0]
+    if admin is 'y':
```

Comment:
Please don't use "is" here. At least my linter complains. Use ==

Reply:
Yep, nice input! Changed